applications/FluidDynamicsApplication/custom_utilities/cfd_utilities.h

```diff
@@ -0,0 +1,283 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Suneth Warnakulasuriya
+//
+
+#if !defined(KRATOS_CFD_UTILITIES_H_INCLUDED)
+#define KRATOS_CFD_UTILITIES_H_INCLUDED
+
+// System includes
+#include <string>
+
+// External includes
+
+// Project includes
+#include "containers/array_1d.h"
+#include "geometries/geometry.h"
+#include "geometries/geometry_data.h"
+#include "includes/define.h"
+#include "includes/model_part.h"
+
+namespace Kratos
+{
+namespace CFDUtilities
+{
```

Comment:
To follow what we normally do, I'd create a `WallLawUtilities`, `WallUtilities`, `CFDWallUtilities` or whatever name you prefer class with static methods to follow what we normally do. If there is an specific reason to implement these as a `namespace` we can discuss about it.

Reply:
I disagree, to me using namespaces aka free functions is much cleaner
I.e. when using a class one never knows if e.g. something is saved internally or not etc
With free functions this is very clear
Plus it follows what the standard library does

I think using classes with static methods is a relic bcs we didn’t know a better way
Esp the py-exposure was a problem I think, but now it is solved